docs/source/building_an_experiment.rst

```diff
@@ -124,22 +124,25 @@ Here's a theoretical example showing all types of interfacing. In this example,
    :alt: An example showing all types of slice interfacing 
    :align: center
 
----------------------
+
 Writing an Experiment
----------------------
+=====================
 
 All experiments must be written as their own class and must be built off of the built-in ExperimentPrototype class.
 
 This means the ExperimentPrototype class must be imported at the start of the experiment file::
 
     from experiments.experiment_prototype import ExperimentPrototype
 
-Experiment-Wide Attributes
---------------------------
+Experiments must not be named starting the 'test' as this is reserved for code debugging purposes
```

Comment:
I took this check for 'test' out actually, now it just doesn't check the 'testing_archive' directory where all the tests are.